staging/cse/windows/nvidiagpudriverfunc.ps1

```diff
@@ -0,0 +1,337 @@
+function Start-InstallGPUDriver {
+  $result = @{
+    RebootNeeded = $false
+  }
+
+  if (-not $PSScriptRoot) {
+    $PSScriptRoot = Split-Path $MyInvocation.InvocationName
+  }
+  if ( -not $env:Path.Contains( "$PSScriptRoot;") ) {
+    $env:Path = "$PSScriptRoot;$env:Path"
+  }
+
+  try {
+    $FatalError = @()
+
+    $LogFolder = "$PSScriptRoot\.."
+
+    $Reboot = @{
+      Needed = $false
+    }
+
+    Write-Host "Attempting to install Nvidia driver..."
+
+    # Get the SetupTarget based on the input
+    $Setup = Get-Setup
+    $SetupTarget = $Setup.Target
+    $Reboot.Needed = $Setup.RebootNeeded
+    Write-Host "Setup complete"
+
+    Add-DriverCertificate $Setup.CertificateUrl
+    Write-Host "Certificate in store"
+
+    Write-Host "Installing $SetupTarget ..."
+    try {
+      $InstallLogFolder = "$LogFolder\NvidiaInstallLog"
```

Comment:
I think that we need to collect the installation logs in 
https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/debug/collect-windows-logs.ps1
https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/provisioningscripts/loggenerator.ps1